lib/rabbitmq/cli/command_behaviour.ex

```diff
@@ -29,7 +29,9 @@ defmodule RabbitMQ.CLI.CommandBehaviour do
                       scopes: 0,
                       usage_additional: 0,
                       switches: 0,
-                      aliases: 0
+                      aliases: 0,
+                      required_rabbit_app_state: 2,
+                      validate_execution_environment: 2
```

Comment:
I would like to see `validate_execution_environment` take the `offline` switch into account in the function's implementation, rather than in the `rabbitmqctl.ex` file.